tests/test_configmanager.py

```diff
@@ -133,24 +136,23 @@ def setUp(self):
 
     def test_parse(self):
         self.configmanager.parse()
-        self.assertIn('eth0', self.configmanager.ethernets)
-        self.assertIn('bond6', self.configmanager.bonds)
+        state = self.configmanager.np_state
+        assert state
+        self.assertIn('eth0', state.ethernets)
+        self.assertIn('bond6', state.bonds)
         self.assertIn('eth0', self.configmanager.physical_interfaces)
-        self.assertNotIn('bond7', self.configmanager.interfaces)
+        self.assertNotIn('bond7', self.configmanager.all_defs)
         self.assertNotIn('bond6', self.configmanager.physical_interfaces)
-        self.assertNotIn('parameters', self.configmanager.bonds.get('bond5'))
-        self.assertIn('parameters', self.configmanager.bonds.get('bond6'))
-        self.assertIn('wwan0', self.configmanager.modems)
+        self.assertIn('wwan0', state.modems)
         self.assertIn('wwan0', self.configmanager.physical_interfaces)
-        self.assertIn('apn', self.configmanager.modems.get('wwan0'))
-        self.assertIn('he-ipv6', self.configmanager.tunnels)
+        # self.assertIn('apn', self.configmanager.modems.get('wwan0'))
+        self.assertIn('he-ipv6', state.tunnels)
         self.assertNotIn('he-ipv6', self.configmanager.physical_interfaces)
-        self.assertIn('remote', self.configmanager.tunnels.get('he-ipv6'))
-        self.assertIn('other-config', self.configmanager.openvswitch)
-        self.assertIn('ports', self.configmanager.openvswitch)
-        self.assertEquals(2, self.configmanager.version)
-        self.assertEquals('networkd', self.configmanager.renderer)
-        self.assertIn('fallback', self.configmanager.nm_devices)
```

Comment:
We're dropping some tests here, but those are just testing some arbitrary values, that are not exposed via libnetplan's API. So that should be fine.